docs/source/index.rst

```diff
@@ -58,7 +58,7 @@ and run a simple client/server example:
 
     $ gcc examples/ucp_client_server.c -lucp -lucs -o ucp_client_server \
           -Iinstall/include -Linstall/lib
-    $ export LD_LIBRARY_PATH=$PWD/instal/lib
+    $ export LD_LIBRARY_PATH=$PWD/install/lib
```

Comment:
i think lines 62-63 should start from ```./bin/ucp_client_server```

Reply:
it will not work, since line 59 writes the output binary in current directory

Reply:
@mike-dubman ok to merge?

Reply:
+1